landing-pages/site/data/committers.json

```diff
@@ -17,6 +17,12 @@
     "image": "/external/profiles/BasPH.png",
     "nick": "BasPH"
   },
+  {
+    "name": "Bu\u011fra \u00d6zt\u00fcrk",
```

Comment:
Ah, looks like pre-commit updated this. its supposed to be `Buğra Öztürk`

Reply:
I am okay with Bugra Ozturk. I am not sure if excluding this file can cause problems in the future

Reply:
I think it should be fine - it's just utf-8 encoded but it should be nicely displayed :)

Reply:
Nice, that would be amazing :)